deployments/sail.dockerfile

```diff
@@ -19,4 +19,4 @@ RUN make build-js
 ADD . .
 RUN make install-sail
 
-ENTRYPOINT sail --web-mode=precompiled
+ENTRYPOINT sail --web-mode=prod
```

Comment:
if we're doing web-mode prod, we probably don't need to include the JS anymore? can get rid of the `make build-js` line above

Reply:
oh shoot this was just for testing and I left it in